datasets/comonscreens.yaml

```diff
@@ -0,0 +1,31 @@
+Name: Common Screens
+Description: A corpus of web screenshot and metadata data composed of over 70 million websites.
+Documentation: https://commonscreens.com/?page_id=1492
+Contact: admin@commonscreens.com
+ManagedBy: "[Common Screens](https://commonscreens.com/)"
+UpdateFrequency: Monthly
+Tags:
+  - aws-pds
+  - encyclopedic
+  - natural language processing
+  - internet
+License: This data is available for anyone to use subject to rules and regulations of USA.
+Resources:

```

Comment:
Please also add the CloudFront distribution info, similar to https://github.com/awslabs/open-data-registry/blob/b1682a37d059754a18d1663d4245f8dc74728b90/datasets/proj-datum-grids.yaml#L13-L16.